std/plugins/test.ts

```diff
@@ -0,0 +1 @@
+import "./plugin_filename_test.ts";
```

Comment:
This is new... I think we only add a `test.ts` for a `mod.ts` with its own code. If the `mod.ts` is just re-exporting stuff we don't need a `test.ts` mirroring it because the test runner already supports directories and such.